src/router.ats

```diff
@@ -66,8 +66,8 @@ class Router {
    * @description Navigate to a URL.
    * Returns the cannonical URL for the route navigated to.
    */
-  navigate(url) {
-    if (this.navigating) {
+  navigate(url, force) {
```

Comment:
Could probably remove `force` if comparing toInstruction and previousInstruction deeply.